Source/Resources/en.lproj/Localizable.strings

```diff
@@ -9,6 +9,7 @@
 "billingFormPhoneNumberHint" = "Phone Number";
 "billingAddressTitle" = "Billing address";
 "BillingFormCountryButtonTitle" = "Please select a country";
+"BillingFormOptional" = "Optional";
```

Comment:
so for now these strings are for only english as we have got them from Alex ( designer ). we will then add more languages support, if we don't provide localisation for other languages but try to run the app in other language or region they are defaulted to english, Nothing would break we just dont support, but with New Design changes we don't need to release a new SDK all the time to support languages , merchants can inject the localised string themselves by using our customisation options.